roles/nvidia-docker/tasks/main.yml

```diff
@@ -0,0 +1,72 @@
+- name: "[nvidia-docker] Install GCCv10"
+  package:
+    name: gcc10.x86_64
+    state: present
+
+- name: "[nvidia-docker] Save previos GCC binary"
+  copy:
+    remote_src: True
+    src: /bin/gcc
+    dest: /bin/gcc-legacy # TODO: old version in name?
+    mode: 755
+- file:
+    path: /bin/gcc
+    state: absent
+
+- name: "[nvidia-docker] Set GCCv10 the default one via symlink"
+  file:
+    src: /bin/gcc10-gcc
+    dest: /bin/gcc
+    state: link
+
+- name: "[nvidia-docker] Remove nouveau kernel module"
+  modprobe:
+    name: nouveau
+    state: absent
+
+- name: "[nvidia-docker] Disable nouveau kernel module"
+  copy:
+    dest: /usr/lib/modprobe.d/nvidia-installer-disable-nouveau.conf
+    content: |
+      blacklist nouveau
+      options nouveau modeset=0
+
+- name: "[nvidia-docker] Download nvidia-{{ nvidia_driver_version }} driver"
+  get_url:
+    url: "https://us.download.nvidia.com/XFree86/Linux-x86_64/{{ nvidia_driver_version }}/NVIDIA-Linux-x86_64-{{ nvidia_driver_version }}.run"
+    dest: "/root/NVIDIA-{{ nvidia_driver_version }}.run"
+
+- name: "[nvidia-docker] Install nvidia-{{ nvidia_driver_version }} driver"
+  shell: "sh /root/NVIDIA-{{ nvidia_driver_version }}.run --silent"
+  register: nvidia_install_log
+
+- name: "[nvidia-docker] Install nvidia-{{ nvidia_driver_version }} log"
+  debug: 
+    msg: "{{ nvidia_install_log.stdout_lines }}"
+
+- name: "[nvidia-docker] nvidia-docker repo"
```

Comment:
Yep, let's use yum_repository instead of shell scripting here since this isn't super idempotent

Reply:
done!